llvm/test/tools/llvm-ir2vec/embeddings.ll

```diff
@@ -0,0 +1,73 @@
+; RUN: llvm-ir2vec --mode=embeddings --ir2vec-vocab-path=%S/../../Analysis/IR2Vec/Inputs/dummy_3D_nonzero_opc_vocab.json %s | FileCheck %s -check-prefix=CHECK-DEFAULT
```

Comment:
Given you access `%S../../Analysis/IR2Vec/Inputs` so often, I'm wondering if it makes sense to add a lit substitution. Maybe something like `%ir2vec_test_vocab` or something.